apiaxle-proxy does not install with nodejs 1.4 or 1.5 on CentOS 7.

I get several errors trying to install apiaxle under CentOS 7
```
npm install -g apiaxle-proxy
```
```
npm WARN deprecated lodash@2.2.1: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
...snip...
In file included from ../src/libxmljs.cc:3:0:
/usr/lib/node_modules/apiaxle-proxy/node_modules/libxmljs/.node-gyp/4.4.3/include/node/v8.h:7603:16: note: static v8::Local<v8::Boolean> v8::Boolean::New(v8::Isolate*, bool)
 Local<Boolean> Boolean::New(Isolate* isolate, bool value) {
                ^
/usr/lib/node_modules/apiaxle-proxy/node_modules/libxmljs/.node-gyp/4.4.3/include/node/v8.h:7603:16: note:   candidate expects 2 arguments, 1 provided
../src/libxmljs.cc:137:19: error: ‘NewSymbol’ is not a member of ‘v8::String’
       target->Set(v8::String::NewSymbol("libxml"), target);
                   ^
../src/libxmljs.cc: In function ‘v8::Handle<v8::Value> libxmljs::ThrowError(const char*)’:
../src/libxmljs.cc:97:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
make: *** [Release/obj.target/xmljs/src/libxmljs.o] Error 1
make: Leaving directory `/usr/lib/node_modules/apiaxle-proxy/node_modules/libxmljs/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 3.10.0-327.10.1.el7.x86_64
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/lib/node_modules/apiaxle-proxy/node_modules/libxmljs
gyp ERR! node -v v4.4.3
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok
npm ERR! Linux 3.10.0-327.10.1.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "-g" "apiaxle-proxy"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE

npm ERR! hiredis@0.1.17 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hiredis@0.1.17 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the hiredis package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs hiredis
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls hiredis
npm ERR! There is likely additional logging output above.
```

--- a/api/package.json
+++ b/api/package.json
@@ -21,7 +21,7 @@
   "dependencies": {
     "lodash": "~1.2.0",
     "apiaxle-base": "1.12.35",
-    "libxmljs": "~0.7.1",
+    "libxmljs": "0.14.x",
     "moment": "~2.0.0",
     "scarf": "0.0.10",
     "sinon": "~1.6.0",

--- a/base/package.json
+++ b/base/package.json
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "lodash": "~1.2.0",
-    "js2xml": "apiaxle/js2xml#v1.0.3",
+    "js2xml": "^1.0.4",
     "libxmljs": "0.14.x",
     "date-utils": "~1.2.13",
     "scarf": "0.0.10",
@@ -25,7 +25,7 @@
     "express": "~3.2.6",
     "async": "~0.2.9",
     "redis": "^0.10.3",
-    "hiredis": "^0.1.17"
+    "hiredis": "^0.4.1"
   },
   "devDependencies": {
     "twerp": "~1.0.7",

